docs/compiler/Introduction.md

```diff
@@ -0,0 +1,57 @@
+<!-- README Document on how to build and use this Neural Network Compiler -->
```

Comment:
this kind of comments only exist in this file. are you planning to put this kind of comments to all other documents?

Reply:
These comments are helpful when somebody updates this md file in future.

Reply:
For the consistancy with other documents, what is your plan. are you going to change all others? or only for this one?
if you are going to add only for this one, I recomment please do not add.

Reply:
I have no plan to add retrospectively. But I plan to have on all documents I add. If you want consistency between existing documents and new ones then I should remove it.